src/operators/startWith.ts

```diff
@@ -0,0 +1,7 @@
+import Observable from '../Observable';
+import ScalarObservable from '../observables/ScalarObservable';
+
+export default function startWith<T>(x: T): Observable<T> {
+  const init = ScalarObservable.just(x);
+  return init.concat(null, <Observable<T>>this);
```

Comment:
@saneyuki thanks for the PR! In the interest of modularity, would you mind updating this PR to pull in the module for the operator you're relying on here, like [`concat`](https://github.com/ReactiveX/RxJS/blob/master/src/operators/concat.ts#L1) does? Since we're aiming for complete modularity here, we can't assume `concat` will exist on the `ScalarObservable` prototype. I'd be satisfied with this PR if you pulled in either of the concat or merge modules, it's your choice. Thanks!